public/app/plugins/panel/stat/StatPanel.tsx

```diff
@@ -82,9 +91,22 @@ export class StatPanelUnconnected extends PureComponent<Props, State> {
     ) {
       this.linkUrl = this.panel.fieldConfig.overrides[0].properties[0].value[0].url;
       this.linkTitle = this.panel.fieldConfig.overrides[0].properties[0].value[0].title;
+      this.expandVariables(this.linkUrl, this.linkTitle);
```

Comment:
Code is ok. Because of repeated code I looked around and thought the code would be more readable if the if conditions were meade into desriptively named functions like hasPanelLinks, hasFieldConfigLinks and hasFieldConfigOverrides.

Deciphering the if conditions raised a question in my mind: is the if sequence here correct - overrides usually implicates that it overrides defaults?

Reply:
Good point: the if sequence should be reversed. Also the idea about descriptive function names is good. I'll do the changes.